src/yaml-language-support/schema-utils.ts

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+import { KubernetesGroupKindNode, KubernetesResourceObjectBase } from "./schema-models";
+
+/**
+ * An utility function for splitting text into lines.
+ */
+export function splitLines(text: string): string[] {
+    return text.split(/\r\n?|\n/);
+} 
+
+/**
+ * A very easy utility function for decides whether the text is support to be a kubernetes yaml file, further
```

Comment:
Comment reads a bit oddly - suggest e.g. "A simple function for deciding whether a piece of text is a Kubernetes YAML file."